backend/src/hatchling/constants.py

```diff
@@ -0,0 +1,3 @@
+from __future__ import annotations
```

Comment:
Let's move this file under `backend/src/hatchling/builders/`

Reply:
Ah, I should've checked. I didn't realize there was already a constants module.